app-foundation/3-networks/development/README.md

```diff
@@ -0,0 +1,51 @@
+# 3-networks/development
+
+This is an additional Terraform configuration for [CFT 3-Networks/envs/development](https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/3-networks/envs/development) and can be used to set up the subnets, additional firewall rules, Cloud Armor, private service networking, and an external IP address for the Bank of Anthos example application in the development environment.
+
+## Prerequisites
+
+1. 0-bootstrap executed successfully.
+1. 1-org executed successfully.
+1. 2-environments/envs/development executed successfully.
+1. 3-networks/envs/shared executed successfully.
+1. Navigate out of the 3-networks/envs/shared repo `cd ..`.
+1. Navigate into the [3-Networks/envs/development](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/3-networks/envs/development) repo `cd development`.
+1. The two local variables - `environment_code = "dev"` and `base_private_service_cidr = "199.36.153.4/30"` - defined in this example are changed in [3-Networks/envs/development/main.tf](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/3-networks/envs/development/main.tf).
```

Comment:
Can we align with example-foundation and just have `environment_code=d`?

Reply:
Sure, I'll change those - the only local variable change would be `base_private_service_cidr` in that case